Content.Server/Weapons/Ranged/Systems/GunSystem.cs

```diff
@@ -303,6 +319,16 @@ private void OnBallisticPrice(EntityUid uid, BallisticAmmoProviderComponent comp
 
     private void ShootOrThrow(EntityUid uid, Vector2 mapDirection, Vector2 gunVelocity, GunComponent gun, EntityUid gunUid, EntityUid? user)
     {
+        if (gun.Target != null && TryComp<FixturesComponent>(uid, out var fixtures))
+        {
+            foreach (var (key, fixture) in fixtures.Fixtures)
+            {
+                Physics.SetCollisionMask(uid, key, fixture, (int) CollisionGroup.LayingDownMobMask);
+                var target = EnsureComp<TargetedProjectileComponent>(uid);
+                target.Target = gun.Target;
+            }
+        }
+
```

Comment:
Bulldozing all of a projectile's collision mask is going to cause bugs, at best you should be adding.

Reply:
The projectile now gets a value added to it's layer that's stored in the StandingStateComponent of the entity